src/main/java/com/microsoft/sqlserver/jdbc/SQLServerError.java

```diff
@@ -7,10 +7,96 @@
 
 import java.io.Serializable;
 
+
 /**
  * SQLServerError represents a TDS error or message event.
  */
 public final class SQLServerError extends StreamPacket implements Serializable {
+
+    /**
+     * List SQL Server transient errors drivers will retry on from
+     * https://docs.microsoft.com/en-us/azure/azure-sql/database/troubleshoot-common-errors-issues#transient-faults-connection-loss-and-other-temporary-errors
+     */
+    enum TransientError {

```

Comment:
Why is 49920 not being handled as a transient error? It looks like 40020 should've been added on line 31, and 40020 should be 49920 instead according to the docs.